libs/designer/src/lib/ui/panel/templatePanel/panel.less

```diff
@@ -152,6 +163,12 @@
                 }
             }
         }
+
+        .msla-template-overview-markdown {
+            * {
```

Comment:
Does this rule need to be a wildcard or can we define this more explicitly?

Reply:
This is to remove the padding for the markdown library element. Adding the styling to the classname itself was not removing the vertical padding/margin, and through research I found out it's the html elements inside the markdown (internal) is causing the issue. I was unable to find out what specific element it would be since it's a third party library, but since the classname msla-template-overview-markdown is used specifically for the markdown, this shouldn't cause errors in other styling